snapcraft_legacy/plugins/v2/kernel.py

```diff
@@ -1425,6 +1425,12 @@ def _add_snappy_ppa(self) -> None:
 
             # add ppa itself
             logger.warning("adding ppa:snappy-dev/image to handle initrd builds")
+            subprocess.run(
```

Comment:
This one will need `sudo`

Reply:
we assume root when running, so should be fine. That said, we should not be adding PPAs through the plugin

Reply:
Can´t this be added to the plugin's build-packages list?